app/src/main/kotlin/com/wire/android/ui/common/Dimensions.kt

```diff
@@ -0,0 +1,7 @@
+package com.wire.android.ui.common
+
+import androidx.compose.ui.unit.dp
+
+object Dimensions {
```

Comment:
```suggestion

@Suppress("MagicNumber")
object Dimensions {
```

Not sure if Detekt will complain about it now, but it definitely will.